src/patch_parse.c

```diff
@@ -74,7 +74,10 @@ static int parse_advance_expected(
 	return 0;
 }
 
-#define parse_advance_expected_s(ctx, str) \
+#define parse_advance_expected_str(ctx, str) \
+	parse_advance_expected(ctx, str, strlen(str))
+
+#define parse_advance_expected_const_str(ctx, str) \
```

Comment:
Yeah, I think that's reasonable.  I think that any compiler that anybody is likely to use will convert `strlen("const")` to a constant and for the person using MSVC 6.0, this isn't likely to be the slow part of reading patches.